source/crud/crud.rst

```diff
@@ -1540,7 +1538,7 @@ Q: Where is write concern?
   However, it might be that a driver needs to expose write concern to a user per operation for various reasons. As noted before, it is permitted to specify this, along with other driver-specific options, in some alternative way.
 
 Q: How do I throttle unacknowledged writes now that write concern is longer defined on a per operation basis?
```

Comment:
is **no** longer?

Reply:
Fixed in https://github.com/mongodb/specifications/pull/208/commits/5d2cc4c5887c911d4d00b5c6fadf2dcd3a89fe7a